lms/djangoapps/course_api/tests/test_api.py

```diff
@@ -144,11 +144,34 @@ def test_for_honor_user_as_anonymous(self):
         with self.assertRaises(PermissionDenied):
             self._make_api_call(anonuser, self.staff_user)
 
+    @SharedModuleStoreTestCase.modifies_courseware
```

Comment:
No, but it does modify data in the modulestore by creating a new course. Since this is a `SharedModuleStoreTestCase`, that second course would then be exposed to other tests. If a test touches the modulestore as part of a `SharedModuleStoreTestCase`, it should clean up after itself to avoid surprises in other tests.